docs/storage/snippets/tutorial/AspireStorage/AspireStorage.AppHost/Program.cs

```diff
@@ -0,0 +1,22 @@
+﻿using Microsoft.Extensions.Hosting;
+
+var builder = DistributedApplication.CreateBuilder(args);
+
+var storage = builder.AddAzureStorage("Storage");
+
+if (builder.Environment.IsDevelopment())
+{
+    storage.UseEmulator();
+}
```

Comment:
This isn't needed, you can just always call UseEmulator

Reply:
That feels weird, and it will be confusing to anyone reading the code. For example, anyone reading this tutorial that hadn't selected Azurite, will see code that reads `UseEmulator`. Don't we want to tie this setup to dev envs?

If it's safe to always call `UseEmulator` why not have that happen when we call `AddAzureStorage`, I'm sure there's reasons?